src/api/app/views/webui/projects/label_templates/index.html.haml

```diff
@@ -11,3 +11,11 @@
           = link_to(edit_project_label_template_path(@project, label_template)) do
             %i.fas.fa-edit
             Edit
+          = link_to(project_label_template_path(@project, label_template), method: :delete,
+            data: { confirm: 'Are you sure you want to delete this label template?'}) do
+            %i.fas.fa-times-circle.text-danger
+            Delete
```

Comment:
```suggestion
          = link_to(project_label_template_path(@project, label_template), method: :delete, class: 'text-danger',
            data: { confirm: 'Are you sure you want to delete this label template?'}) do
            %i.fas.fa-trash-can
            Delete
```

Reply:
Fixed

Reply:
Actually, technically, it should be `link-danger` https://getbootstrap.com/docs/5.3/utilities/link/#colored-links